AudiusProject/audius-protocol
# [CON-112] Add worker to check if track is available

### Description



This PR's goal is to add a worker to discovery to check track availability in the network. 

This PR includes:
- a worker in discovery to check track availability across content nodes
- tests for the worker

### Tests



- See `discovery-provider/integration_tests/tasks/test_update_track_is_available.py`
- Also manually tested
1. Upload 3 tracks
2. Mark track id 1 as unavailable on all content nodes, track id 2 as available on all nodes, track id 3 only available on 1 node
3. Ensure the following:
```
track id 1: is_available = false
track id 2: is_available = true
track id 3: is_available = true
```

### How will this change be monitored? Are there sufficient logs?
- There will be redis keys with the prefix `update_track_is_available:start` and `update_track_is_available:finish` where we can check the timestamp of when the worker jobs started and finished
- also redis sets with the same prefix `update_track_is_available:unavailable_tracks_<all or sp id>` to show you which track ids are unavailable in redis if need to debug
- logs w the prefix `update_track_is_available.py` in the worker
- updated health check response with latest start/finish timestamps

health check response example
```
   "last_track_unavailability_job_end_time": "2022-06-06 21:38:57.069988 UTC",
    "last_track_unavailability_job_start_time": "2022-06-06 21:38:57.040422 UTC",
```
